docs/kb/api-version-3/export.md

```diff
@@ -140,7 +140,7 @@ Output Representation
 ### Errors
 
 * Status Code: 4xx
-* Error codes: See [Error Codes](/kb/error-codes-messages)
+* Error codes: See [Error Codes](../error-codes-messages.md)
```

Comment:
The error codes link seems to work already. This change may not be needed. Also the extra *.md may generate an invalid link

Reply:
Just the opposite; the link with the ".md" is proper way to reference links to other pages within the site and the "/kb/" link may break in the future.